### ISSUE TITLE
extraneous "!" in the text?

### ISSUE THREAD

**Issue by Li-YangZhong** [TimothyGu/es-howto#13]
The "!" is shown in the section below:
********************************
`When Boolean is called with argument value, the following steps are taken:

Let b be ! ToBoolean(value).

If NewTarget is undefined, return b.

Let O be ? OrdinaryCreateFromConstructor(NewTarget, "%BooleanPrototype%", « [[BooleanData]] »).

Set O.[[BooleanData]] to b.

Return O.`
********************************

However, the following section seems to indicate that there is not a "!" in the section above and the ECMA-262(2019 edition) don't have a "!" in there either. I think the "!" should be removed.

********************************
`Step 1 casts value (the function argument) to a Boolean value. Interestingly there isn’t a ? or ! shorthand for this step, but usually not having a Completion Record shorthand means the same thing as !. So step 1 cannot throw an exception.`
********************************

**Comment by ExE-Boss** [comment]
[The `!` shorthand](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) is present in <https://tc39.es/ecma262/#sec-boolean-constructor-boolean-value>.

**Comment by TimothyGu** [comment]
Indeed. There has been recently a few clean-ups on the usage of these shorthands in the spec. See https://github.com/tc39/ecma262/pull/1622 and https://github.com/tc39/ecma262/pulls?q=label%3A%22completion+records%22.

**Comment by TimothyGu** [comment]
That's a long winded way of saying: I'd appreciate a pull request :)

**Comment by uaparicio** [comment]
I'm a beginner contributor and I was looking for easy issues to fix when I found this one. I can take care of the ! removal (In spite I don't know what are you talking about there)

**Comment by uaparicio** [comment]
https://github.com/TimothyGu/es-howto/pull/14
